benchmark/README.md

```diff
@@ -0,0 +1,71 @@
+# looks-same Benchmark
+
+This benchmark evaluates the performance of four npm packages for image comparison: [`looks-same`](https://github.com/gemini-testing/looks-same), [`pixelmatch`](https://github.com/mapbox/pixelmatch), [`resemblejs`](https://github.com/rsmbl/Resemble.js), and [`blink-diff`](https://github.com/yahoo/blink-diff).  
```

Comment:
blink-diff is archived. Should we still use it?

Reply:
The repo is archived, but the package is not deprecated: https://www.npmjs.com/package/blink-diff